modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -97,6 +100,9 @@
  * Raft server implementation on top of forked JRaft library.
  */
 public class JraftServerImpl implements RaftServer {
+    /** Enables logit log storage. {@code true} by default. */
+    public static final String LOGIT_STORAGE_ENABLED_PROPERTY = "LOGIT_STORAGE_ENABLED";
```

Comment:
Is using a system property a temporary solution?

Reply:
Yes, it is temporary, by the time we release the product, only one log storage will be left. I'll say that in the comment